scripts/hack/symlink-gopath-codebuild.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# This script is meant to make all package available in the 'canonical' location
+# for execution in AWS CodeBuild in the golang container image.
+
+
+if [[ ! -d "/go/src/github.com/awslabs/amazon-ecr-credential-helper" ]]; then
```

Comment:
I'm not familiar with where codebuild places the build directory, is this hack a workaround for the way that this is dir is mounted into the building container?

Reply:
Yeah, it's a workaround to make builds work properly on forks.  CodeBuild seems to place the source in a directory like `/codebuild/output/src<number>/src/github.com/<username>/<repo>` and then append that to the `GOPATH`, so the first time I tried building this I got errors trying to locate packages in the `GOPATH`.

This is pretty much the same thing we used to do for TravisCI before they added the `go_import_path` field.